tests/hooks/use-pinheader.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from "bun:test"
+import { Circuit } from "lib/Circuit"
+import { usePinHeader } from "lib/hooks/use-pinheader"
+
+test("usePinheader hook creates component with correct props", () => {
+  const circuit = new Circuit()
+
+  const useHeader = usePinHeader("Header", {
+    pinLabels: ["VCC", "GND", "TX", "RX"],
+  })
+
+  const Header = useHeader("Header", {
+    pinCount: 4,
+    pinLabels: ["VCC", "GND", "TX", "RX"],
+  })
```

Comment:
The usage is wrong, see the useChip or useResistor examples.


const J1 = usePinHeader(“J1”, { pinLabels })

J1.myPin // string

<J1 />

Reply:
gotcha, I've fixed the variable names 👍🏻